metrics/web/lib/dashboard/presentation/state/project_metrics_notifier.dart

```diff
@@ -396,14 +396,30 @@ class ProjectMetricsNotifier extends ChangeNotifier {
       );
     }
 
-    final maxBuildDuration = _getMaxBuildDuration(buildResults);
+    const numberOfBuildsToDisplay =
+        ReceiveProjectMetricsUpdates.buildsToLoadForChartMetrics;
+    final buildsCount = buildResults.length;
 
-    final buildResultViewModels =
-        buildResults.map(_createBuildResultViewModel).toList();
+    final lastBuildIndex = buildsCount <= numberOfBuildsToDisplay
+        ? 0
+        : (buildsCount - numberOfBuildsToDisplay);
+
+    final latestBuildResults = buildResults.sublist(lastBuildIndex);
```

Comment:
```suggestion
    List<BuildResult> latestBuildResults = buildResults;

    if (buildsCount <= numberOfBuildsToDisplay) {
      latestBuildResults = latestBuildResults.sublist(
        buildsCount - numberOfBuildsToDisplay,
      );
    }
```
Would it be a bit more readable?